pkg/api/validation/validation.go

```diff
@@ -2844,6 +2854,8 @@ func ValidateService(service *api.Service) field.ErrorList {
 		allErrs = append(allErrs, field.NotSupported(specPath.Child("sessionAffinity"), service.Spec.SessionAffinity, supportedSessionAffinityType.List()))
 	}
 
+	allErrs = append(allErrs, validateClientIPAffinityConfig(service.Spec.ClientIPAffinityConfig, specPath.Child("clientIPAffinityConfig"))...)
```

Comment:
also we should probably document and enforce an upper bound.  86400 for 1 day?  604800 for 1 week?